scrappers/models/Promotion.py

```diff
@@ -0,0 +1,39 @@
+from scripts.utils import brand_by_name_get, product_by_name_get
+
+
+class Promotion:
+    def __init__(self, store_name, begin_date, end_date, name, price, promo):
+        self.store_name = store_name
+        self.beginDate = begin_date
+        self.endDate = end_date
+        self.name = name
+        self.price = float(price.replace(",", ".")) if "," in price else float(price)
+        self.promotion = float(promo.replace(",", ".")) if "," in promo else float(promo)
+
+    def __str__(self):
+        return "Store: {} / Product name: {} / Product price: {} / Promotion: {}" \
```

Comment:
You could use format strings 
```suggestion
        return f'Store: {self.store} / Product name: {self.name} / Product price: {self.price} / Promotion: {self.promo}'
```